package.json

```diff
@@ -14,28 +14,32 @@
     "2d",
     "mobile"
   ],
+  "author": "Ali Shakiba",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/shakiba/planck.js.git"
   },
-  "main": "dist/planck.js",
-  "types": "index.d.ts",
-  "author": "Ali Shakiba",
   "contributors": [
     "Erin Catto",
     "Ali Shakiba",
     "Oliver Zell"
   ],
-  "dependencies": {
-    "stage-js": "^0.9.4"
+  "type": "module",
+  "types": "./index.d.ts",
+  "main": "./dist/planck.umd.cjs",
+  "module": "./dist/planck.js",
+  "browser": "./dist/planck.cjs",
```

Comment:
I think the browser field is useless in our case because node.js and the browsers should use the same file (and cjs is probably wrong?)